integrations/segment/hub.md

```
@@ -0,0 +1,48 @@
+
+- **Track Event**
+  - Tracks a custom event and associated properties in Segment Analytics.
+  - Inputs: User ID, Event Name, Event Payload (optional JSON string of event properties)
+
+### Configuration
+
+- **Write Key**: The write key is required to authenticate requests to Segment Analytics. After creating a Source you can generate a Write Key by going to Sources -> API Keys -> Settings -> Write Key
+
+## Setup Instructions
+
+#### Segment Analytics Setup
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)